src/service/resources-impl.js

```diff
@@ -506,7 +508,8 @@ export class Resources {
     const parentResource = Resource.forElement(parentElement);
     subElements = elements_(subElements);
 
-    this.discoverResourcesForArray_(parentResource, subElements, resource => {
+    this.discoverResourcesForArray_(parentResource, subElements,
+    /* deepSubtree */ true, resource => {
```

Comment:
Shouldn't the parent be in charge of unlayout-ing it's children? We have it being responsible for the layout.